pkg/extensions/extension_search.go

```diff
@@ -63,7 +66,7 @@ func downloadTrivyDB(log log.Logger, updateInterval time.Duration) error {
 }
 
 func SetupSearchRoutes(config *config.Config, router *mux.Router, storeController storage.StoreController,
-	log log.Logger,
+	searchDB repodb.RepoDB, log log.Logger,
```

Comment:
we are calling repoDB type as searchDB var? why?

Reply:
Will rename it.